plugins/modules/vmware_object_rename.py

```diff
@@ -46,6 +46,34 @@
       required: True
       aliases: ['object_new_name']
       type: str
+    filter:
+      description:
+      - Filter an object to search.
+      - This module can't get over 1,000 objects at onece, so you should use the I(filter) parameter
```

Comment:
```suggestion
      - This module can not get over 1,000 objects at once, so you should use the I(filter) parameter
```

Reply:
Note: the new REST API of vSphere 7.0.2 does not have this limitation.